libslide/Cargo.toml

```diff
@@ -5,3 +5,4 @@ authors = ["ayazhafiz <ayaz.hafiz.1@gmail.com>"]
 edition = "2018"
 
 [dependencies]
+overload = "0.1.1"
```

Comment:
Also, I added a dependency which is just a list of macros to shorten operator overloading. Should be very lightweight.

TBH, I would rather us not have any dependencies for this project (DIY kind beat). If needed we can implement overload! ourselves. What do you think?